pages/projects/index.js

```diff
@@ -19,19 +21,33 @@ export default function Projects({ projects }) {
         Lorem ipsum ad tempor ea elit anim aliqua nulla exercitation veniam id non elit in in deserunt magna.
       </Typography>
 
-      {
-        projects.map(project => (
-          <Fragment key={ project.slug }>
-            <Link to={ `/projects/${ project.slug }` }>
-              { project.name }
-            </Link>
-            <Pre>
-              { JSON.stringify(project, null, 2) }
-            </Pre>
-          </Fragment>
-        ))
-      }
+      <Typography paragraph>
+        Learn more about each project at RENCI below. 
+      </Typography>
+
+      <Box sx={{
+        flex: 1,
+        marginTop: '3rem',
+        display: 'grid',
+        gap: '2rem',
+        gridTemplateColumns: 'repeat(auto-fit, 250px)',
+      }}>
+        {
+          projects.map((project) => {
+            const trimText = (description, wordCount = 27) => {
+              const snippetArray = (description || 'Click to read more').split(' ')
+              const trimmedSnippetArray = snippetArray.slice(0, wordCount)
+              return snippetArray.length >= wordCount ? `${trimmedSnippetArray.join(' ')} ...` : trimmedSnippetArray.join(' ')
+            }
+            const snippet = trimText(project.description)
 
+            return (
+              <SpotlightCard project={project} key={`spotlight-${project.slug}`} snippet={snippet}/>
```

Comment:
since we'd now be using this component in more places than just the spotlight, i wonder if the name "SpotlightCard" isn't quite right anymore

Reply:
great point! I think ProjectCard would be more fitting now!